src/semeio/fmudesign/create_design.py

```diff
@@ -1020,9 +1013,9 @@ def _printwarning(corrgroup):
         "have to be included in correlation matrix. \n "
         "See documentation: \n"
         "https://equinor.github.io/fmu-tools/"
```

Comment:
"fmu-tools" should probably not be here?

Reply:
Actually, the docs are still in `fmu-tools` for now. Not been decided where to put them yet, so will leave this as is for now. But thanks.